src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const interArr = sourceString.split(';');
+
+  const arr = interArr
+    .filter((item) => item.trim().length > 0)
+    .map((item) => item.trim().replace(/\s*:\s*/g, ':'));
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have same indentation. This makes code much more readable. Also, [CODE KNOWLEDGE] - Don't need to make copy before each operation. The `filter` method does not mutate the original array, so you can chain it directly without creating a separate variable `interArr`.